File: src/main/java/com/uwetrottmann/trakt/v2/entities/ListIds.java
package com.uwetrottmann.trakt.v2.entities;

public class ListIds {

    public Integer trakt;
    public String slug;

    public static ListIds trakt(int trakt) {
        ListIds ids = new ListIds();
        ids.trakt = trakt;
        return ids;
    }

    public static ListIds slug(String slug) {
        ListIds ids = new ListIds();
        ids.slug = slug;
        return ids;
    }

}

File: src/main/kotlin/com/redissi/trakt/entities/ListIds.kt
package com.redissi.trakt.entities

import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
class ListIds(
    val trakt: Int? = null,
    val slug: String? = null
) {

    companion object {
        fun trakt(trakt: Int): ListIds {
            return ListIds(trakt = trakt)
        }

        fun slug(slug: String?): ListIds {
            return ListIds(slug = slug)
        }
    }
}